libbpf-rs/src/link.rs

```diff
@@ -15,6 +16,17 @@ impl Link {
         Link { ptr }
     }
 
+    /// Create link from BPF FS file.
+    pub fn open<P: AsRef<Path>>(&mut self, path: P) -> Result<Self> {
+        let path_c = util::path_to_cstring(path)?;
+        let path_ptr = path_c.as_ptr();
+        let ptr = unsafe { libbpf_sys::bpf_link__open(path_ptr) };
+        if ptr.is_null() {
```

Comment:
Currently, the libbpf 1.0 mode is not enabled by default. You should use `libbpf_get_error` here.

Reply:
I can change this. 

When will we be using libbpf 1.0 btw? I figured that given the milestone we'd be using that version.

Reply:
The libbpf library isn't on 1.0 yet (so libbpf-rs can't using 1.0).

This is determined here: https://github.com/libbpf/libbpf/blob/6edaacad4f7c31f08f63205e2b3569b9f946c8cb/src/libbpf_internal.h#L474 
https://github.com/libbpf/libbpf/blob/3591deb9bc6b0848721f831790b78ae0a593b4c5/src/libbpf.c#L154-L155